kettle/monitor.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I'm mildly weirded out by having py2/py3 co-mingled in kettle, since none of the other files have been converted.  Should we install `python-pip` and `pip install influxdb` in the image to support running this instead?

Reply:
That is a valid concern. Of the various options, I went with this one mainly because, as @Katharine pointed out, `monitor.py` was not used anywhere else within `kettle`. Another alternative is (1); I almost have the PY2 -> PY3 changes ready for `kettle`, so we can wait and add those to this PR? All options are viable. 

Options:
1. full migration of kettle to PY3
2. __partial migration of kettle (`monitor.py`)__
3. image compatibility w/ PY2/PY3